### PR TITLE
Add code of conduct notices

### PR DESCRIPTION
**Pull request by seanbudd**
Add notice that add-ons should comply with the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/registerAddon.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ body:
 - type: markdown
   attributes:
     value: |
-      Submit an NVDA add-on to datastore. Provide download and source URL, and indicate if it's a stable add-on or not.
+      Submit an NVDA add-on to datastore.
+      Add-ons should comply with the the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```
Comment (michaelDCurran) [inline .github/ISSUE_TEMPLATE/registerAddon.yml:14]: Fix typo (extra 'the'):
```suggestion
      Add-ons should comply with the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
+However, the metadata for a new submission (add-on release) can be confirmed to match its manifest description.
 Additionally, add-on file integrity can be enforced via a SHA256 checksum.
 The checksum allows NVDA to ensure that add-on releases are immutable.
 
 ### Human review process / code audit
 - NV Access doesn't require a manual review of the add-on (code or user experience) itself before the add-on submission.
 - You are welcome to review code / UX of add-ons and provide that feedback directly to add-on authors.
 - The SHA256 checksum of the `.nvda-addon` prevents undetected changes.
-- User reviews/rating of add-ons are currently out of scope.
+- Add-ons should comply with the the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```
Comment (michaelDCurran) [inline README.md:25]:
```suggestion
- Add-ons should comply with the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```

File: docs/submitters/submissionGuide.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Submission Guide
 If your add-on was hosted on [addonFiles](https://github.com/nvaccess/addonFiles) please read the [migrating to datastore guide](./migratingFromAddonFiles.md).
 
+Submitted add-ons should comply with the the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md)
```
Comment (michaelDCurran) [inline docs/submitters/submissionGuide.md:4]:
```suggestion
Submitted add-ons should comply with the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md)
```

**Review by michaelDCurran — APPROVED** [review]
With suggested typo fixes, this is good.

**Comment by nvdaes** [comment]
I wonder if a complementary note about add-on ID ijacking, and if it's or not acceptable to submit add-ons without consulting original authors, should be here, to avoid the creation of a list of add-ons submitters,which, as pointed out by @XLTechie in issue #1165 , may have important drawbacks.

**Comment by seanbudd** [comment]
@nvdaes - I think it's more appropriate to include that when #1165 is addressed

**Pull request merged**